pkg/frontend/openshiftcluster_preflightvalidation.go

```diff
@@ -9,10 +9,10 @@ import (
 	"net/http"
```

Comment:
Generic question. Why do have this flat structure in the FE directory? A lot of files have a prefix that makes me think they should be in a sub-package... What am I missing?

Reply:
no reason, feel free to propose a refactor if you'd like :)